tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -39,14 +41,18 @@ bool IsRetriable(Status status) {
 
 Status CallWithRetries(const std::function<Status()>& f) {
   int attempts = 0;
+  int delay;
```

Comment:
The compiler will take care of optimization if it is beneficial. Please do make the change.